src/lib.rs

```diff
@@ -101,6 +101,19 @@ impl<K: Hash + Eq + Clone + 'static> TabView<K> {
         self.current_id.as_ref()
     }
 
+    pub fn get_active_tab(&self) -> Option<&Box<dyn View>> {
```

Comment:
Do you need to expose the `Box` layer here? Returning `Option<&dyn View>` may fit most use-cases, is a bit simpler, and keeps the actual storage internal (so it'd be easy to move to `Rc` or other without breakage).

Reply:
Yep, sounds like a good idea. I'll wait on the discussion in #15 tho, maybe this PR gets superseded by another more general rework of the API. :small_airplane: :smile:

Reply:
A good idea :D We should do this to allow more choices for the user of the library. Also, with the discussion at https://github.com/deinstapel/cursive-tabs/issues/15 we decided that this PR will be included :D
I'll add a few commits to update the PR to reflect the changes proposed here and then add the PR.